Assets/Scripts/Interactables/Dialogue/NPCDialogue/NPCInteractable.cs

```diff
@@ -359,6 +369,21 @@ public void NPCFacePlayer()
         StartCoroutine(RotateAnimation(gameObject, rotation, GameManager.Instance.GetMovementController().turnSpeed));
     }
 
+    public void NPCFaceOriginalRotation(Conversation finishedConvo)
+	{
+        if (finishedConvo != conversation) return;
+
+        DialogueManager.Instance.OnFinishConversation -= NPCFaceOriginalRotation;
+        StopAllCoroutines();
+        StartCoroutine(RotateToOriginal(1.0f));
+	}
+
+    private IEnumerator RotateToOriginal(float waitTime)
+	{
+        yield return new WaitForSeconds(waitTime);
+        StartCoroutine(RotateAnimation(gameObject, originRotation, GameManager.Instance.GetMovementController().turnSpeed));
```

Comment:
I think this turn speed is for the player... just realized it's a bit weird to use this for NPC rotations...
`GameManager.Instance.GetMovementController().turnSpeed`
if it works maybe it's fine just to get this in though LOL

Reply:
Yeah it like snaps back haha which is really strange! But we used that for the turn speed of facing towards so the player so I just used that speed as well. Not hard to change though!